dm/pkg/terror/error_list.go

```diff
@@ -897,6 +899,8 @@ var (
 		"config '%s' regex pattern '%s' invalid, reason: %s", "Please check if params is correctly in the configuration file.")
 	ErrConfigOnlineDDLMistakeRegex = New(codeConfigOnlineDDLMistakeRegex, ClassConfig, ScopeInternal, LevelHigh,
 		"online ddl sql '%s' invalid, table %s fail to match '%s' online ddl regex", "Please update your `shadow-table-rules` or `trash-table-rules` in the configuration file.")
+	ErrOpenAPITaskConfigExists    = New(codeConfigOpenAPITaskConfigExists, ClassConfig, ScopeInternal, LevelLow, "the openapi task config  for '%s' already exists", "If you want to override it, pleaset use the overwite flag.")
```

Comment:
```suggestion
	ErrOpenAPITaskConfigExists    = New(codeConfigOpenAPITaskConfigExists, ClassConfig, ScopeInternal, LevelLow, "the openapi task config for '%s' already exists", "If you want to override it, pleaset use the overwite flag.")
```

and check typo

Reply:
@Ehco1996 pleaset -> please

Reply:
this sentences is genereaed by github copilot and my ide plugin does not check for syntax errors 😂